Remote control guide updates to improve document flow (#515)
smartdevicelink/sdl_app_library_guides: docs/Other SDL Features/Remote Control Vehicle Features/index.md

!@

#### Getting Module Data Location and Service Areas (RPC v6.0+)

> With the saved remote control capabilities struct you can get the location of the each module and the area that it services. This will map to the `grid` [graphic](#the-grid) below.  This information is useful for creating a custom UI. 

Reviewer: ```suggestion
With the saved remote control capabilities struct you can get the location of the each module and the area that it services. This will map to the `grid` [graphic](#the-grid) below. This information is useful for creating a custom UI.
```

```
!@

> You can also get an array of seats in the `seatLocationCapability`'s `seat` array. Each @![iOS]`SDLSeatLocation`!@@![android, javaSE, javaEE, javascript]`SeatLocation`!@ object within the `seats` array will have a `grid` parameter. The `grid` will tell you the seat placement in the vehicle of that particular seat (See the [graphic](#the-grid) below).

Reviewer: ```suggestion
You can also get an array of seats in @![iOS]`SDLSeatLocationCapability.seats`!@@![android, javaSE, javaEE, javascript]`SeatLocationCapability.seats!@ array. Each @![iOS]`SDLSeatLocation`!@@![android, javaSE, javaEE, javascript]`SeatLocation`!@ object within the `seats` array will have a `grid` parameter. The `grid` will tell you the location of that particular seat in the vehicle (See the [graphic](#the-grid) below).
```

```
!@

> An array of seats can be found in the `seatLocationCapability`'s `seat` array. Each @![iOS]`SDLSeatLocation`!@@![android, javaSE, javaEE, javascript]`SeatLocation`!@ object within the `seats` array will have a `grid` parameter. The `grid` will tell you the seat placement of that particular seat. This information is useful for creating a seat location map from which users can select their seat.

Reviewer: ```suggestion
An array of seats can be found in the `seatLocationCapability`'s `seat` array. Each @![iOS]`SDLSeatLocation`!@@![android, javaSE, javaEE, javascript]`SeatLocation`!@ object within the `seats` array will have a `grid` parameter. The `grid` will tell you the seat placement of that particular seat. This information is useful for creating a seat location map from which you can determine module service areas, and from which users can select their seat (see the [Setting the User Seat](#setting-the-users-seat-rpc-v6.0) section).
```

I'm not sure if that link is correct or not.

```
!@

> ### The Grid

Reviewer: ```suggestion
#### The Grid
```

### Setting Module Data
Not only do you have the ability to get data from these modules, but, if you have the right permissions, you can also set module data.

#### Setting The User's Seat (RPC v6.0+)

> Before you attempt to take control of any module, you should have your user select their seat location as this affects which modules they have permission to control. You may wish to show the user a map or list of all available seats in your app in order to ask them where they are located. See [Getting Module Data Location and Service Areas](#getting-module-data-location-and-service-areas-rpc-v60) for information useful in creating a custom UI showing module location and service area.  The following example is only meant to show you how to access the available data and not how to build your UI/UX. 

Reviewer: ```suggestion
Before you attempt to take control of any module, you should have your user select their seat location as this affects which modules they have permission to control. You may wish to show the user a map or list of all available seats in your app in order to ask them where they are located. See [Getting Module Data Location and Service Areas](#getting-module-data-location-and-service-areas-rpc-v60) for information useful in creating a custom UI showing module location and service area. The following example is only meant to show you how to access the available data and not how to build your UI/UX.
```
